TPCircularBuffer.h

```diff
@@ -143,6 +143,25 @@ static __inline__ __attribute__((always_inline)) void* TPCircularBufferTail(TPCi
     return (void*)((char*)buffer->buffer + buffer->tail);
 }
 
+/*!
+ * Access end of buffer with a given offset
+ *
+ *  This gives you a pointer to the end of the buffer plus a given offset, ready
```

Comment:
*  This gives you a pointer to the end of the buffer plus a given offset, ready
```suggestion
 *  This gives you a pointer to the tail of the buffer plus a given offset, ready
```